src/affexpr.jl

```diff
@@ -112,6 +112,8 @@ function value(a::GenericAffExpr{T, V}, map::Function) where {T, V}
     ret
 end
 
+constant(aff::GenericAffExpr) = aff.constant
```

Comment:
Is this meant to be a public API or internal only? If it's the former it should have a doc string, if it's the latter then it could use a scarier name like `extract_constant` and comment that it's meant for internal use.